spec/unit/packagers/rpm_spec.rb

```diff
@@ -252,7 +251,7 @@ module Omnibus
 
       it 'uses the correct command' do
         expect(subject).to receive(:shellout!)
-          .with(/rpmbuild -bb --buildroot/)
+          .with(/rpmbuild --target x86_64 -bb --buildroot/)
         subject.create_rpm_file
       end
```

Comment:
I think I probably should be stubbing out safe_architecture here to ensure that the target is always x86_64 in this test case.